content/sensu-core/0.29/guides/troubleshooting.md

```diff
@@ -97,6 +97,68 @@ Sensu staff, or community members may ask to see your logs. You can view them at
 {{< highlight shell >}}
 tail -n 10000 /var/log/sensu/sensu-server.log > sensu-server-10k.log && gzip -9 sensu-server-10k.log{{< /highlight >}}
 
+### Troubleshooting via the Local Client Socket
+This checklist is intended to be a step-by-step walkthrough of how to troubleshoot Sensu via the local client socket. There will be various examples for you to use and build on as you troubleshoot. 
+
+#### Local Client Socket
```

Comment:
Since we're trying to keep this guide as short as possible, I'd recommend removing lines 101-103